Dockerfile

```diff
@@ -58,10 +64,37 @@ RUN jupyter labextension install nbdime-jupyterlab --no-build && \
         jupyter lab clean && \
         jlpm cache clean && \
         npm cache clean --force && \
-        rm -rf $HOME/.node-gyp && \
-        rm -rf $HOME/.local && \
-    fix-permissions $CONDA_DIR $HOME
+        rm -rf "${HOME}/.node-gyp/" && \
+        rm -rf "${HOME}/.local/" && \
+    fix-permissions "${CONDA_DIR}" "${HOME}"
+
+# READ-ONLY notebooks
```

Comment:
Should they be a read only folder? Often, students just create a copy of the notebook to run in class so that it doesn't conflict with the course version and general github awkwardness